docker-compose.yml

```diff
@@ -47,25 +48,17 @@ services:
       # BAD_HOST: "${BAD_HOST}"
       # OVERIDE_HOST: "${OVERIDE_HOST}"
       # STRATEGY: "${STRATEGY}"
-      SENTENCE_TRANSFORMERS_HOME: "${SENTENCE_TRANSFORMERS_HOME}"
-      HF_HOME: "${HF_HOME}"
-      TRANSFORMERS_CACHE: "${TRANSFORMERS_CACHE}"
-      HF_DATASETS_CACHE: "${HF_DATASETS_CACHE}"
+      #NEO4J_URL: "bolt://neo4j:password@neo4j:7687"
     volumes:
       - ./app:/opt/app/openlxp-xss
-      - huggingface_data:/opt/app/huggingface
     depends_on:
-      - neo4j
```

Comment:
The neo4j dependency should not be removed here, as the app container does definitely depend on it.

Reply:
Uhh, I did not touch the .yml file for the manual search function.